### PR TITLE
[Fix](load)Use lower case for load column names.

### PR DESCRIPTION
**Pull request by Jibing-Li**
# Proposed changes

Issue Number: close #xxx

## Problem summary

The columns name in stream load and broker load are case sensitive, make it case insensitive. This would be consist with query, because query sql columns name are case insensitve.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.09 seconds
 load time: 480 seconds
 storage size: 17122704567 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230206014711_clickbench_pr_90580.html

File: fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java [hunk 1]
```diff
@@ -195,6 +195,8 @@ public DataDescription(String tableName,
         this.deleteCondition = deleteCondition;
         this.sequenceCol = sequenceColName;
         this.properties = properties;
+        columnsNameToLowerCase(fileFieldNames);
```
Comment (carlvinhust2012) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java:198]: if this change take effect, would the column name of query result become low case?

Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java:199]: No. This change only change the user input stream/broker load columns name, will not affect the table columns.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**